lib/spotify.js

```diff
@@ -82,3 +117,13 @@ module.exports = {
   toggleShuffleState,
   ascertainShuffleState,
 };
+
+// probably excessive, but I like the detailed verbose output with --verbose
+module.exports = _.reduce(functions, (acc, value, key) => {
```

Comment:
this and `waitForCondition` could be broken out of this file, but we can wait until we want to use them elsewhere